src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -17,6 +21,13 @@ import type {
 } from '../../types/trezor/protobuf';
 import type { CoreMessage } from '../../types';
 
+// todo: remove when listed firmwares become mandatory for cardanoSignTransaction
+const CardanoSignTransactionFeatures = Object.freeze({
+    SignStakePoolRegistrationAsOwner: ['', '2.3.5'],
```

Comment:
```suggestion
    SignStakePoolRegistrationAsOwner: ['0', '2.3.6'],
```
0 to be consistent with others and `2.3.6` - because 2.3.5 was never released

Reply:
@szymonlesisz - I see changelog still lists the version 2.3.5 as the one to be released on February 10th: https://github.com/trezor/trezor-firmware/blob/44c7d2374168fab867e167095c0e32aa5fdcea39/core/CHANGELOG.md - is that outdated?